tests/test_sites.py

```diff
@@ -117,8 +118,8 @@ def test_get_course_enrollments_for_site(self):
                        [ce.id for ce in expected_ce])
 
 
-@pytest.mark.skipif(not organizations_support_sites(),
-                    reason='Organizations support sites')
+# @pytest.mark.skipif(not organizations_support_sites(),
+#                     reason='Organizations support sites')
```

Comment:
I haven't removed any code from tests. Just made enough changes so that compatibility changes work.

Reply:
No need to keep the commented code. Please remove it.